centreon/www/front_src/src/Dashboards/components/DashboardLibrary/DashboardListing/Actions/favoriteAction/FavoriteAction.tsx

```diff
@@ -25,13 +25,19 @@ interface Props {
   dashboardId: number;
   isFavorite: boolean;
   refetch?: () => void;
+  isFetching: boolean;
 }
 
-const FavoriteAction = ({ dashboardId, isFavorite, refetch }: Props) => {
+const FavoriteAction = ({
+  dashboardId,
+  isFavorite,
+  refetch,
+  isFetching
+}: Props) => {
   const { t } = useTranslation();
   const labelSuccess = useRef('');
   const { showSuccessMessage } = useSnackbar();
-  const [isPending, startTransition] = useTransition();
+  const [, startTransition] = useTransition();
```

Comment:
I am not sure, but can't we optimise this by calling **startTransition** directly from react

https://react.dev/reference/react/startTransition

thank you

Reply:
yess , but depend on the latest note on the doc: "You can call startTransition when useTransition is not available. For example, startTransition works outside components, such as from a data library" . 👀

Reply:
ah, thanks for the info